articles/azure/azs-how-create-S2S-VPN-azurestack-to-azure.md

```diff
@@ -226,27 +234,31 @@ You'll also need to create a virtual network in Public Azure. This virtual netwo
 
 5. In the **Create virtual network** blade, enter the following information:
 
-   - **Name** - The name of the virtual network.
-
-   - **Address Space** - The virtual network's address range in CIDR notation.
-
    - **Subscription** - This is your Microsoft Azure subscription.
 
    - **Resource Group** - Select an existing resource group, or create a new one by typing a name for your new resource group.
 
-   - **Location** - Select the location of the Azure resource.
+   - **Name** - The name of the virtual network.
+
+   - **Region** - Select the location of the Azure resource.
+
+       ![Create new virtual network](images/azs-public-browser-new-virtual-network-blade.png)
+
+   - **Address Space** - The virtual network's address range in CIDR notation. This must not overlap the address space of the Azure Stack Hub virtual network.
 
    - **Subnet Name** - The name of the first subnet within the virtual network.
+   
+   - **Subnet Address Range** - The subnet's address range in CIDR notation (for example, 192.168.1.0). It must be within the address range of the virtual network. It should be noted that the address range of a subnet currently in use cannot be edited.
```

Comment:
To make the last sentence active rather than passive, go for something like:
You cannot edit the address range of a subnet that is currently in use.
(I think you use a similar phrase earlier in the article, so just make sure that you use the same phrase in both places)